myokit/formats/sbml/_importer.py

```diff
@@ -203,89 +291,102 @@ def _parse_parameters(self, model, comp, refs, node):
             else:
                 # Create variable
                 unit = None
-                if node.hasAttribute('units'):
-                    foreign_unit = node.getAttribute('units')
+                if node.get('units'):
```

Comment:
Not sure you need two lookups here!

Reply:
(The old minidom syntax didn't give a nicer way to do this)

Reply:
Agreed. Done.